source/MaterialXGenOgsFx/OgsFxShaderGenerator.h

```diff
@@ -48,6 +48,8 @@ class OgsFxShaderGenerator : public GlslShaderGenerator
 
     /// Get parameters for the technique block
     virtual void getTechniqueParams(const Shader& shader, string& params) const;
+
+    StringMap _semanticsMap;
```

Comment:
Make semantics map a class member instead of global static variable to avoid static initialization order issues.